servlet/theLearningServlet/src/test/java/jakartaee/examples/servlet/thelearningservlet/TheLearningServletTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Permission to use, copy, modify, and/or distribute this software for any 
+ * purpose with or without fee is hereby granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR(S) DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR(S) BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+ * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+package jakartaee.examples.servlet.thelearningservlet;
+
+import com.gargoylesoftware.htmlunit.TextPage;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import jakartaee.examples.utils.JakartaEEServer;
+import java.io.File;
+import java.net.URL;
+import org.arquillian.container.chameleon.runner.ArquillianChameleon;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import static org.jboss.shrinkwrap.api.ShrinkWrap.create;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.After;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * The JUnit tests for the Servlet API @WebServlet example.
+ *
+ * @author Manfred Riem (mriem@manorrock.com)
```

Comment:
Is this correct? :P

Reply:
There are two issues I will resolve. The easier one is to expand the test to return to the index.html page and press the second button. The second one has to do with the email address that I inadvertently used to sign the submission. As I can edit the config locally I'm hoping that when I push it to my repo it will now have the correct user name and email address.